packages/studio-ui/src/web/components/Layout/Sidebar.tsx

```diff
@@ -133,7 +133,8 @@ const Sidebar: FC<Props> = props => {
 
 const mapStateToProps = (state: RootReducer) => ({
   viewMode: state.ui.viewMode,
-  modules: state.modules
+  modules: state.modules,
+  isCloudBot: Boolean(state.bot?.cloud?.clientId)
```

Comment:
one could create a create a cloud ready bot but not enter client ID just yet. That could would return false even though it's a cloud bot. 

Isn't there a boolean set in the bot config ?